src/BookGroup.java

```diff
@@ -15,6 +15,8 @@ public class BookGroup {
     private ArrayList<String> members;
     private ArrayList<GroupPost> posts;
     private ArrayList<Integer> currentPages;
+    //NEW SPRINT3
+    private ArrayList<Boolean> isDone;
```

Comment:
to keep track of if each book buddy has finished the book (the data for a particular user is in the same position/index in all the arrays in BookGroup object)